sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -146,7 +146,7 @@ class ComputeInstance(Compute):
     :type enable_node_public_ip: Optional[bool], optional
     :param setup_scripts: Details of customized scripts to execute for setting up the cluster.
     :type setup_scripts: Optional[SetupScripts], optional
-    :param custom_applications: Describes available applications and their endpoints on this ComputeInstance.
+    :param custom_applications: Experimental. List of custom applications for the compute instance.
```

Comment:
No need of putting experimental in docstring. It should be coming from @decorator

Reply:
I believe, @annatisch meant to add the details about the experimental parameter in the docstring of the class.

`
class ComputeInstance(Compute):
    """Compute Instance resource.

    custom_applications is an experimental parameter.
    """
`

Reply:
This is what I had meant - but you're saying it's already handled?
I would recommend just checking the CI build artifacts to see the rendered docs and checking the parameter is documented correctly. If so, then it should be fine.

Reply:
The parameter appears to be documented correctly here: 
![image](https://user-images.githubusercontent.com/90645783/216644969-161a5244-a2ef-4964-bc92-11eaaba99bcc.png)

Reply:
I noticed from my previous screenshott that SetupScripts was also not public, so I have taken the liberty of adding that in as well:

![image](https://user-images.githubusercontent.com/90645783/216660865-9ac1e86a-2e68-476e-9806-a0f0093b3c8d.png)